strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/pom.xml

```diff
@@ -138,6 +138,11 @@
             <groupId>org.awaitility</groupId>
             <artifactId>awaitility</artifactId>
         </dependency>
+        <dependency>
+    		<groupId>com.googlecode.json-simple</groupId>
+    		<artifactId>json-simple</artifactId>
+    		<version>1.1.1</version>
+		</dependency>
```

Comment:
We already have other libraries for JSON, such as Jackson. How does this one help and can't you use Jackson for this instead?
Also, this is not indented as per our coding convention.